CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ModeSolverData.pol_fraction` and `ModeSolverData.pol_fraction_waveguide` properties to compute polarization fraction of modes using two different definitions.
 - `ModeSolverData.to_dataframe()` and `ModeSolverData.modes_info` for a convenient summary of various modal properties of the computed modes.
 - Loss upper bound estimation in `PoleResidue` material model.
+- `tidy3d/converter.py` for conversion from .lsf project files into equivalent python scripts implementing the project in Tidy3D through the command line, e.g. `tidy3d convert example.lsf tidy3d_script.py`
```

Comment:
just noticed this. I think the file is no longer here right? so I think we can just say
```
Command line tool `tidy3d convert` for conversion from `.lsf` project files into equivalent python scripts implementing the project in Tidy3D. Usage:  `tidy3d convert example.lsf tidy3d_script.py.
```

Reply:
Nice catch, implemented